packages/models-library/src/models_library/clusters.py

```diff
@@ -32,7 +34,7 @@ class Config:
 class SimpleAuthentication(BaseAuthentication):
     type: Literal["simple"] = "simple"
     username: str
-    password: str
+    password: SecretStr
```

Comment:
Below the ``JupyterHubTokenAuthentication.api_token`` should also be a SecretStr

Reply:
hmm are you sure? that is already a token. and anyway I think we might disregard all this jupyterhub/kerberos stuff